### PR TITLE
fix tematica checkboxes

### PR DESCRIPTION
**Pull request by d2bit**
todas should return publicado only
* cache must expire updating any record
* todas should cache an array not a query

### CODE REVIEW

**Comment by scruti** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54ec63c5695680565e0016e4/compare/fix-tematicas-checkbox">analyzed this pull request</a>.

File: app/concerns/tematica/tematica_base.rb [hunk 1]
```diff
@@ -8,24 +6,33 @@ module Tematica::TematicaBase
...
     def nombre(id)
-      Rails.cache.fetch("tematica-nombre-#{id}") do
+      Rails.cache.fetch("tematica-nombre-#{ id }") do
         Tematica::Tematica.find(id).nombre
       end
     end
 
     def todas
       Rails.cache.fetch("tematicas") do
-        Tematica::Tematica.all
```
Comment (Fernan2) [inline app/concerns/tematica/tematica_base.rb]: No me jodas...

**Comment by Fernan2** [comment]
+1, much better now! :clap:

**Comment by scruti** [comment]
:+1:

**Comment by gponsu** [comment]
:+1:

**Pull request merged**